apps/web/src/components/shared/ProfileImage.tsx

```diff
@@ -0,0 +1,72 @@
+import { IconOutlineFileUpload, IconPerson, Text } from '@novu/design-system';
+import React from 'react';
+import { Control } from 'react-hook-form';
+import { cx } from '../../styled-system/css';
+
+import { imageUploadStyles, previewContainerStyles } from './ProfileImage.styles';
+import useProfileImageForm from './useProfileImageForm';
+type ProfileImageProps = {
+  /**
+   * The URL of the image to display
+   */
+  imageUrl?: string;
+  /**
+   *
+   * @param image The image file to submit
+   * @returns A promise that resolves when the image is submitted
+   */
+  onImageSubmit: (image: File | null) => Promise<void>;
+
+  /**
+   * The react-hook-form control object
+   */
+  control: Control<any>;
+  /**
+   * The name of the image input field that is used in react-hook-form
+   */
+  name: string;
+};
+
+type UploadInputContainerProps = Pick<ProfileImageProps, 'control' | 'onImageSubmit' | 'name'>;
+
+export function ProfileImage({ onImageSubmit, imageUrl, control, name }: ProfileImageProps) {
```

Comment:
This component should accept: `ref, name, value, onChange` props which will be passed by the parent component, that will wrap it with the `Controller` or `useController` hook.